### PR TITLE
Add admin panels for all models

### PR DESCRIPTION
**Pull request by mikix**
https://openedx.atlassian.net/browse/AA-93

### CODE REVIEW

File: edx_when/admin.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    def field(self, obj):
+        return obj.content_date.field
+
+    def date(self, obj):
+        return obj.abs_date or obj.rel_date
+
+    def save_model(self, request, obj, form, change):
+        obj.actor = request.user
+        super().save_model(request, obj, form, change)
```
Comment (mikix) [inline edx_when/admin.py:63]: Is this appropriate for the `actor` field @cpennington, or should I not try to be clever and just leave `actor` in place when editing via the admin form?

Comment (cpennington) [reply edx_when/admin.py]: I'm not sure where that `actor` field is coming from. Is that from the history model?

Comment (mikix) [reply edx_when/admin.py]: No, it's a top level field defined specifically in `UserDate` -- I assume it's fillled from the user that publishes in Studio?

Comment (mikix) [reply edx_when/admin.py]: Er, not publishes, but overrides the user schedule rather. Same person that fills in `reason` I guess.

Comment (cpennington) [reply edx_when/admin.py]: Ah, ok. Yeah, I'd record the current admin user, then.

**Pull request merged**